src/cmd/group.go

```diff
@@ -38,9 +39,161 @@ EOF
 	},
 }
 
-// TODO: Get Group
+var getGroupCommand = &cobra.Command{
+	Use:        "group ID|ALIAS",
+	Short:      "Get details about a group",
+	Long:       `Get details about a group`,
+	Args:       cobra.ExactArgs(1),
+	ArgAliases: []string{"ID", "ALIAS"},
+	Run: func(cmd *cobra.Command, args []string) {
+		key := args[0]
+		var group *opslevel.Group
+		var err error
+		if common.IsID(key) {
+			group, err = getClientGQL().GetGroup(key)
+		} else {
+			group, err = getClientGQL().GetGroupWithAlias(key)
+		}
+		cobra.CheckErr(err)
+		common.PrettyPrint(group)
+		common.WasFound(group.Id == nil, key)
+	},
+}
 
-// TODO: List Groups
+var getMembersCommand = &cobra.Command{
```

Comment:
lets call this `getGroupMembersCommand` because its nested within the group subcommand